### PR TITLE
[luci-interpreter]Negative Tests for Add kernel

### PR DESCRIPTION
**Pull request by struss**
For #1623 

This commit add negative tests on `Add` kernel on luci-interpreter.

ONE-DCO-1.0-Signed-off-by: KiDeuk Bang <rrstrous@nate.com>

### CODE REVIEW

File: compiler/luci-interpreter/src/kernels/Add.test.cpp [hunk 1]
```diff
@@ -169,6 +169,33 @@ TEST(AddTest, Float)
...
+{
+  Tensor input1_tensor = makeInputTensor<DataType::FLOAT32>({1}, {1.f});
+  Tensor input2_tensor = makeInputTensor<DataType::S32>({1}, {2});
+  Tensor output_tensor = makeOutputTensor(DataType::FLOAT32);
+
+  AddParams params{};
+  params.activation = Activation::RELU;
+
+  Add kernel(&input1_tensor, &input2_tensor, &output_tensor, params);
+  EXPECT_ANY_THROW(kernel.configure());
```
Comment (struss) [inline compiler/luci-interpreter/src/kernels/Add.test.cpp:182]: @llFreetimell On assert condition is not caught using `EXPECT_ANY_THROW`. Could I replace `EXPECT_ANY_THROW` into `EXPECT_DEATH` with assert?

Comment (seanshpark) [reply compiler/luci-interpreter/src/kernels/Add.test.cpp:182]: CI test passed.  Can you explain more about your concern? 

> assert condition

you've changed from `assert` to `throw`. what do you mean by `assert condition` ?

Comment (struss) [reply compiler/luci-interpreter/src/kernels/Add.test.cpp:182]: Oh, currently All Input validation case and some invalid value catch is made up with `assert`. so, I have 2 option, change into throw or keep that and use `EXPECT_DEATH` as a checker.

Comment (seanshpark) [reply compiler/luci-interpreter/src/kernels/Add.test.cpp:182]: `assert` will work differently with release build. let's use `throw` with testing.

**Pull request merged**